### PR TITLE
Updates VRC

### PR DESCRIPTION
**Pull request by PatrickGoettsch**

### CODE REVIEW

**Comment by timothy-wiley** [comment]
The picture for clarity is nice.
The updates the the comments appear to be consistent along with small language improvements.

File: content/technical_challenges.tex [hunk 1]
```diff
@@ -116,117 +116,146 @@ \subsection{Visual Referee Challenge}
...
+                \item Using its arm(s) to \emph{mirror} the referee's signal. That is, if the referee gives a signal for a decision for the ``red team'' using their left arm, the robot should \emph{mirror} this signal using the robot's \emph{right arm}, as the robot is facing the referee.
                 \item Providing an audio phrase the referee's decision, e.g., ```Goal Red Team'''.  The exact wording is up to the team, but should clearly identify the referee's signal.
-                \item (Optional), At the discretion of the GC/TCM developers, TCP messages may also be sent and interpreted to display the outcome of the robot's decision, however, this is not a requirement for the execution of the challenge.
+                \item The robots output should be repeated as long as, it gets a signal by the referee pressing a button to indicate the time for the next signal. This should give the referee time to understand the robot's audio output clearly.
```
[the remark this answers is not in the document]
Comment (PatrickGoettsch) [reply content/technical_challenges.tex]: I specified a button. thanks for bringing this up

**Review by rvilling — COMMENTED** [review]
The new figure is very clear.

The figure suggests that the robot will be placed on the center spot (as I originally assumed). But the text only says that the robot will be placed "in the center circle". Is it intentional that the robot can be moved around inside the circle?

Spelling: "The referee must wear a ``black-and-white'' stripped referee jersey" --> "The referee must wear a ``black-and-white'' striped referee jersey"

What does the button do? Does the button cause the robot to repeat its output (arm and speech), or must the robot repeat its output continuously until the button stops its output (arm and speech)? 

A robot "may continue to pose in the same position..." If it chooses not to continue to pose in this position, can it just show the arm motion for a few seconds (e.g. 5 seconds)?

Minor corrections: "The _robot's_ output [...] as long as [deleted comma] its front head button [...]"

"The referee should choose 2 one-hand static signals, 2 two-hand static signals, and 1 dynamic signal." I think that there are only 2 two-handed static signals and 1 dynamic signal so it is clear which signals these will be. So the only real choice (2 out of 3) is for one hand static signals.

**Comment by PatrickGoettsch** [comment]
Also Rudi's last questions are now addressed

File: content/technical_challenges.tex [hunk 2]
```diff
@@ -146,7 +146,7 @@ \subsection{Visual Referee Challenge}
             \begin{enumerate}
                 \item Using its arm(s) to \emph{mirror} the referee's signal. That is, if the referee gives a signal for a decision for the ``red team'' using their left arm, the robot should \emph{mirror} this signal using the robot's \emph{right arm}, as the robot is facing the referee.
                 \item Providing an audio phrase the referee's decision, e.g., ```Goal Red Team'''.  The exact wording is up to the team, but should clearly identify the referee's signal.
-                \item The robots output should be repeated as long as, it gets a signal by the referee pressing a button to indicate the time for the next signal. This should give the referee time to understand the robot's audio output clearly.
+                \item The robots output should be repeated as long as, its front head button gets pressed for at least \qty{1}{\second} to indicate that the next signal will come. This should give the referee time to understand the robot's audio output clearly.
```
Comment (ahasselbring) [inline content/technical_challenges.tex]: :thumbsup:

**Pull request merged**